src/testSession.ts

```diff
@@ -58,22 +60,22 @@ export interface TestSessionOptions {
  *   TESTKIT_JWT_KEY = JWT key (not a filepath, the actual contents of the key)
  *   TESTKIT_HUB_INSTANCE = instance url for the hub.  Defaults to https://login.salesforce.com
  *   TESTKIT_AUTH_URL = auth url to be used with auth:sfdxurl:store
- 
  */
+
 export class TestSession {
   public id: string;
   public createdDate: Date;
   public dir: string;
   public homeDir: string;
   public project?: TestProject;
-  public setup?: AnyJson[] | shell.ShellString;
+  public setup?: ExecCmdResult[];
+  public orgs: string[] = [];
```

Comment:
this was only meant for org deletion purposes in the clean function, that's why it was private.  I think it's confusing/unnecessary to provide this and `setup` results for use within tests.

Reply:
@shetzel My setup includes creation of two scratch orgs, one set with default username and the other with --nonamespace. The org usernames names are captured, but if I want to access those I need to tear apart the setup array in order to do so. Seemed convenient to just expose these here, but I am not against using the results to get these.

Reply:
I agree it's convenient but discussing with TD a few weeks ago we decided that it wasn't worth making org create setup commands special in this way and tests should just access setup commands via `session.setup` for simple consistency.  This would prevent accidental mutation of the orgs array as well which is used for session cleanup. Getting more opinions on this is a good thing though so others feel free to weigh in.

Reply:
@shetzel I will revert making orgs public.